exonum/src/blockchain/config.rs

```diff
@@ -220,6 +222,15 @@ impl StoredConfiguration {
             ));
         }
 
+        // Check storage_version.
+        if config.storage_version != storage::StorageVersion::current() {
```

Comment:
I'm not sure I follow this logic of version verification. Is it possible to change the storage version using the configuration service? If so, what should that signal? I'd imagine it would (indirectly) signal the full nodes to upgrade their software in order to support the new version; i.e., once a new config is activated, the node should stop if it does not support the new storage version (and perhaps a do some kind of migration otherwise before continuing to work). I'm not sure that this check reflects this kind of logic.

Reply:
I'm not sure that user should be able to initiate such migration. I think that it should be performed for example as part of migration from 1.0 to 2.0.